launch/tier4_perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml

```diff
@@ -67,7 +67,7 @@
   <group if="$(eval &quot;'$(var lidar_detection_model)'=='centerpoint'&quot;)">
     <push-ros-namespace namespace="centerpoint"/>
     <group>
-      <include file="$(find-pkg-share lidar_centerpoint)/launch/lidar_centerpoint.launch.xml">
+      <include file="$(find-pkg-share lidar_centerpoint_tvm)/launch/lidar_centerpoint_tvm.launch.xml">
```

Comment:
Are we back to using the TVM package by default? I don't know if it will work so well for end users in the current state of things. The download of compiled models is disabled by default, so the lidar based detection will not be working out of the box in the pre-compiled image for example. And locally people will need to know that they need to set `DOWNLOAD_ARTIFACTS` as well as setting the backend to OpenCL. Also we still don't have runtime performance figures against cuda. Especially for the C++ pre/post processing.

Reply:
Sry, I always forget to reset this.